keystore-manager.sh

```
@@ -282,6 +353,771 @@ encode_base64() {
+
+    # Scan secrets directory for known files
+    for file_name in "${!FILE_TO_SECRET_MAP[@]}"; do
+        local file_path="secrets/$file_name"
+        local secret_name="${FILE_TO_SECRET_MAP[$file_name]}"
+
+        if [ -f "$file_path" ]; then
+            secrets_found=$((secrets_found + 1))
+            echo -e "${BLUE}Found: $file_name${NC}"
+            echo -e "${BLUE}Encoding as: $secret_name${NC}"
+
+            local encoded=$(encode_base64 "$file_path")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)